new-from-ui.go

```diff
@@ -0,0 +1 @@
+package main
```

Comment:
The file has increased in 1 lines.
<!-- lookout footnote separator -->
_Comment made by the analyzer Dummy._